### PR TITLE
Update release notes for 2.0.0

### PR DESCRIPTION
**Pull request by WhyNotHugo**
Some minor missing entries.

### CODE REVIEW

**Review by davidhalter — APPROVED** [review]

**Comment by WhyNotHugo** [comment]
setuptools-scm doesn't work with Python 3.6. We should drop that one too; it's past its EOL.

File: docs/release-notes.rst [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+2.0.0
 ----------
 * New: Support for Django 3.2 and 4.0
 * Dropped Django <3.2 support.
 * New: Support for Python 3.9 and 3.10.
 * Moved CI to GitHub Actions.
 * Dropped Python 3.5 support.
+* Add detection for Chromium-based Edge browser
+* Rename OS X to macOS
+* Add detection for newere macOS versions
```
Comment (claudep) [inline docs/release-notes.rst:13]: newere -> newer?

Comment (WhyNotHugo) [reply docs/release-notes.rst]: :+1:

**Comment by WhyNotHugo** [comment]
CI is all green!

File: docs/release-notes.rst [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 =============
 
-Unreleased
+2.0.0
 ----------
 * New: Support for Django 3.2 and 4.0
 * Dropped Django <3.2 support.
 * New: Support for Python 3.9 and 3.10.
 * Moved CI to GitHub Actions.
 * Dropped Python 3.5 support.
```
Comment (claudep) [inline docs/release-notes.rst:10]: Just noticed that we also dropped 3.6 support.

**Review by claudep — APPROVED** [review]
:+1:

**Pull request merged**